wire-ios-data-model/Tests/MLS/MLSServiceTests.swift

```diff
@@ -1722,3 +1891,30 @@ extension MLSGroupID {
     }
 
 }
+
+extension QualifiedID {
+
+    static func random() -> QualifiedID {
+        return QualifiedID(
+            uuid: .create(),
+            domain: .randomDomain()
+        )
+    }
+
+}
+
+extension String {
+
+    static func randomDomain(hostLength: UInt = 5) -> String {
+        return "\(String.random(length: hostLength)).com"
+    }
+
+    static func random(length: UInt) -> String {
+        let randomChars = (0..<length).compactMap { _ in
+            "a...z".randomElement()
+        }
+
+        return String(randomChars)
+    }
+
+}
```

Comment:
is this only used here ? If so consider use private or move to some utils files instead

Reply:
I moved them to separate files